### ISSUE TITLE
EPUB annotation UX question/issue

### ISSUE THREAD

**Issue by kajdo** [elias-sundqvist/obsidian-annotator#105]
Hi,

maybe I just don't find an option to change, but it seems that there isn't an opiton to tell the epub reader to open in a single page view.

using it on a usual monitor on a desktop the double page layout works, but due to the strange width / height ratio, there is a lot of real estate wasted, which could be used for more epub content

see screenshot as example
![image](https://user-images.githubusercontent.com/8735942/147710483-13112c6a-c411-464d-9788-65f588055549.png)

using a 1920x1080 resolution and dedicate the whole screen for obsidian and the annotator should provide most of the screen for the reader part imho, otherwise someone would need to open an epub reader to read AND obsidian with the annotator to annotate - which would ruin the whole workflow

comparing that with the pdf annotation - everything works fine and most of the space is used for the pdf

if I'm just missing something - pls give me a hint

thx for the great plugin!!! most of the time, I need it for pdfs anyhow :)

**Comment by ondrejdyck** [comment]
I agree. Maybe on a large monitor it is not so noticeable but on my laptop it feels like a tremendous amount of real estate is wasted with padding. Also an option to adjust font size would be handy. 

I have been looking high an low for an epub reader that allows comments, bookmarks etc. where you can go back later and actually find something you read. This one looks very promising.

**Comment by elias-sundqvist** [comment]
I've released a new beta version now (0.2.0) that reduces the spacing of the epub reader. It also allows you to change font size (Thanks to @aladmit for the PR).
You can upgrade to 0.2.0 using [BRAT](https://github.com/TfTHacker/obsidian42-brat).

**Comment by mikhasev** [comment]
Hi, but if you are using epub.js library you could give access to main.css etc. files related to the reader view and anyone could fiddle with columns, spacing, font or even background color. Вщ you think its possible? I certainly did it with calibre-web app. Its epub.js...
